response_operations_ui/templates/collection_exercise/ce-collection-instrument-eq-section.html

```diff
@@ -70,12 +118,13 @@
                       {% endif %}
                   {% endfor %}
               {% endfor %}
-            </div>
+              </div>
+            {% endif %}
             <p id="ci-checked-count">{{ collection_instruments['EQ']|length }} selected out of {{ all_cis_for_survey|length }} available</p>
```

Comment:
I don't know if this is included in this card, but this line is meant to come out as bold in the design

Reply:
Yeah I have followed only the yellow highlighted areas for this and so I've left this as it is.